io.github.nest-desktop.yml

```diff
@@ -0,0 +1,88 @@
+id: io.github.nest-desktop
```

Comment:
did you see the application ID guidelines in PR checklist?

![image](https://github.com/user-attachments/assets/4b8932ae-b5aa-4dbd-aa50-b58b65f1e954)

Reply:
is `org.nest_simulator.nest-desktop` fine when the page exists?

Reply:
Would be `org.nest_simulator.nest-desktop` fine for the page `https://nest-simulator.org/nest-desktop`?

Reply:
The domain belongs to the NEST community. NEST Desktop, a GUI for NEST Simulator, is my contribution to the community.

Reply:
> The domain belongs to the NEST community. NEST Desktop, a GUI for NEST Simulator, is my contribution to the community.

Unless you control it you cannot use it. Use the `io.github.nest_desktop.nest-desktop`

Reply:
Ok, I am renaming it.